service/src/main/java/bio/terra/workspace/service/workspace/flight/CreateCustomGcpRolesStep.java

```diff
@@ -9,14 +9,20 @@
 import bio.terra.stairway.exception.RetryException;
 import bio.terra.workspace.service.resource.controlled.mappings.CustomGcpIamRole;
 import bio.terra.workspace.service.resource.controlled.mappings.CustomGcpIamRoleMapping;
+import bio.terra.workspace.service.workspace.CloudSyncRoleMapping;
 import com.google.api.client.googleapis.json.GoogleJsonResponseException;
 import com.google.api.services.iam.v1.model.CreateRoleRequest;
 import com.google.api.services.iam.v1.model.Role;
+import com.google.common.collect.ImmutableSet;
 import java.io.IOException;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.http.HttpStatus;
 
+/**
+ * This step creates custom role definitions in our GCP context. It does not grant these roles to
+ * any users, though other steps do.
+ */
 public class CreateCustomGcpRolesStep implements Step {
```

Comment:
Aside: for steps that are only meant to be used  in one flight, I wonder if it would help organization to make them static inner classes of the flights themselves. Or adopt a kludgy naming convention like `MyFlight_CreateRolesStep`.

Reply:
For one flight I think we tried making steps like that inner classes inside the flight. It was good for directory organization but got a little messy for nontrivial steps. It also ends up being messier if we're wrong about the step only being part of one flight, since we have to break it out into a separate class. Worth thinking about though